realm/realm-library/src/androidTest/java/io/realm/RealmMigrationTests.java

```diff
@@ -221,68 +228,37 @@ public long execute(Realm realm, long version) {
     }
 
     public void testSetAnnotations() {
+
+        // Create v0 of the Realm
+        RealmConfiguration originalConfig = new RealmConfiguration.Builder(getContext()).schema(AllTypes.class).build();
+        Realm.deleteRealm(originalConfig);
+        Realm.getInstance(originalConfig).close();
```

Comment:
What's the point of this line? After its execution the state is the same as after the previous line.